packages/uiweb/src/lib/components/space/SpacesUI.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import SpaceBanner, { SpaceBannerProps } from './SpaceBanner';
+import SpaceFeed, { SpaceFeedProps } from './SpaceFeed';
+import SpaceWidget, { SpaceWidgetProps } from './SpaceWidget';
+import SpaceTrendingList, { SpaceTrendingListProps } from './SpaceTrendingList';
+import { useSpaceData } from '../../context';
+import { SignerType } from '../../types';
+import { ENV } from '../../config';
```

Comment:
nit: leave a newline after ever level of import. 
helps in better code readability

example: 
```suggestion
import React from 'react';

import SpaceBanner, { SpaceBannerProps } from './SpaceBanner';
import SpaceFeed, { SpaceFeedProps } from './SpaceFeed';
import SpaceWidget, { SpaceWidgetProps } from './SpaceWidget';
import SpaceTrendingList, { SpaceTrendingListProps } from './SpaceTrendingList';

import { useSpaceData } from '../../context';
import { SignerType } from '../../types';
import { ENV } from '../../config';
```